src/apps/loan-list/list/loan-list.entry.tsx

```diff
@@ -61,6 +61,9 @@ export interface LoanListEntryProps {
   bottomDueDateRenewLoanModalCheckboxText: string;
   bottomRenewLoanModalButtonText: string;
   bottomDueDateRenewLoanModalButtonText: string;
+  loanListDigitalLoansEmptyListText: string;
```

Comment:
Hint: If you put the list of texts in alphabetical order you'll make the person that is integrating the app in dpl-cms happy.